modules/economy-system/events/botReady.js

```diff
@@ -26,6 +26,21 @@ module.exports.run = async function (client) {
         client.logger.info('[economy-system] ' + localize('economy-system', 'migration-done'));
         await client.models['DatabaseSchemeVersion'].create({model: 'economy_Cooldown', version: 'V1'});
     }
+    const dbVersionShop = await client.models['DatabaseSchemeVersion'].findOne({where: {model: 'economy_Shop'}});
+    if (!dbVersionShop) {
+        client.logger.info('[economy-system] ' + localize('economy-system', 'migration-happening'));
+        const data = await client.models['economy-system']['Shop'].findAll({attributes: ['name', 'price', 'role']});
+        await client.models['economy-system']['Shop'].sync({force: true});
+        i = 0;
```

Comment:
```suggestion
        let i = 0;
```
variable is only declared implicitly